src/components/listview/listview.js

```diff
@@ -105,7 +105,6 @@ define(['itemHelper', 'mediaInfo', 'indicators', 'connectionManager', 'layoutMan
         var apiClient = connectionManager.getApiClient(item.ServerId);
 
         var options = {
-            width: width,
```

Comment:
`maxWidth: width * 2,`
Potentially, this will be the same as `getImageUrl`: 80px or 500px.
I can’t verify this - need for normal LiveTV tuner.